src/coreclr/src/vm/interpreter.cpp

```diff
@@ -10615,6 +10649,20 @@ void Interpreter::DoSIMDHwAccelerated()
     LdIcon(1);
 }
 
+
+void Interpreter::DoGetIsSupported()
+{
+    CONTRACTL{
+        THROWS;
+        GC_TRIGGERS;
+        MODE_COOPERATIVE;
+    } CONTRACTL_END;
+
+    OpStackSet<BOOL>(m_curStackHt, false);
+    OpStackTypeSet(m_curStackHt, InterpreterType(CORINFO_TYPE_INT));
```

Comment:
The interpreter should be in sync on which hardware intrinsics are supported. This should return the same value as if the method was JITed.

Reply:
I think I saw a reply from @jkotas in email but github isn't showing it to me now. We should surround the logic that makes IsSupported return FALSE with a config variable that developers would need to explicitly set + comment that it is only intended for internal testing purposes.

@jkotas suggestion that we invoke the JIT for all of these intrinsics seemed like a reasonable production solution when the time comes in the future. I have some concerns that the high overhead interpreter has to call methods may make the performance look pretty bad, but that is a concern for a later time.

Reply:
I added a `COMPlus` variable `InterpreterHWIntrinsicsIsSupportedFalse` in commit https://github.com/dotnet/runtime/pull/37825/commits/ba907e0fe46657872e7a6e74ec5aa5341ace7224 to force `get_IsSupported` to return false.